spec/support/acceptance/helpers.rb

```diff
@@ -93,7 +93,7 @@ def create_record(fields)
   def get_target_record(target)
     task_result = servicenow_instance.run_bolt_task(
       'servicenow_tasks::get_records',
-      { 'table' => 'cmdb_ci', 'fields' => { 'fqdn' => target.uri } },
+      { 'table' => 'cmdb_ci', 'url_params' => { 'fqdn' => target.uri, 'sysparm_display_value' => true } },
```

Comment:
We also needed this fix with https://github.com/puppetlabs/servicenow_tasks/pull/13 being merged to `servicenow_tasks`

Reply:
This was also needed to make the tests pass on a real ServiceNow instance.